dev/com.ibm.ws.jaxrs.2.1_fat_executor/fat/src/com/ibm/ws/jaxrs/executor/fat/FATSuite.java

```diff
@@ -12,12 +12,22 @@
  *******************************************************************************/
 package com.ibm.ws.jaxrs.executor.fat;
 
+import org.junit.ClassRule;
 import org.junit.runner.RunWith;
 import org.junit.runners.Suite;
 import org.junit.runners.Suite.SuiteClasses;
 
+import componenttest.rules.repeater.JakartaEE10Action;
+import componenttest.rules.repeater.RepeatTests;
+
 @RunWith(Suite.class)
 @SuiteClasses({
                 JAXRSExecutorTest.class
 })
-public class FATSuite {}
\ No newline at end of file
+public class FATSuite {
+    @ClassRule
+    public static RepeatTests r = 
+        RepeatTests.withoutModification()
+                   .andWith(new JakartaEE10Action())
+                   .andWith(new JakartaEE10Action().removeFeature("concurrent-1.0").removeFeature("concurrent-3.0").withID(JakartaEE10Action.ID + "_w/o_concurrent_feature"));
```

Comment:
I wonder if some platforms there will be issues with the `/` character.  It is a file separator for directories and the ID is used in the file names and directory names so I wonder if it will mess things up.  Maybe change it to a different character.

Reply:
changed "w/o" to "without"